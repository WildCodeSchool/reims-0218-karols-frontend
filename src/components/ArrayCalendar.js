import React from "react"
import ListCalendarDayTime from "./ListCalendarDayTime"
import { Table } from "reactstrap"

const ArrayCalendar = () => {
  return (
    <Table responsive>
      <tbody>
        <tr>
          <td>
            <ListCalendarDayTime />
          </td>
          <td>
            <ListCalendarDayTime />
          </td>
          <td>
            <ListCalendarDayTime />
          </td>
          <td>
            <ListCalendarDayTime />
          </td>
        </tr>
      </tbody>
    </Table>
  )
}

export default ArrayCalendar
