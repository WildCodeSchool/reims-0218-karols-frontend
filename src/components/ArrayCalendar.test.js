import React from "react"
import ReactDOM from "react-dom"
import ArrayCalendar from "./ArrayCalendar"
import renderer from "react-test-renderer"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<ArrayCalendar />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<ArrayCalendar />).toJSON()
  expect(tree).toMatchSnapshot()
})
