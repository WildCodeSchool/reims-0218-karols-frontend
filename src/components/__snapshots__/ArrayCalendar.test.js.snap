// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="table-responsive"
>
  <table
    className="table"
  >
    <tbody>
      <tr>
        <td>
          <div>
            <div
              className="text-center"
            >
              <h5>
                Vendredi
              </h5>
              <h5>
                25 mai
              </h5>
            </div>
            <div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
            </div>
          </div>
        </td>
        <td>
          <div>
            <div
              className="text-center"
            >
              <h5>
                Vendredi
              </h5>
              <h5>
                25 mai
              </h5>
            </div>
            <div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
            </div>
          </div>
        </td>
        <td>
          <div>
            <div
              className="text-center"
            >
              <h5>
                Vendredi
              </h5>
              <h5>
                25 mai
              </h5>
            </div>
            <div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
            </div>
          </div>
        </td>
        <td>
          <div>
            <div
              className="text-center"
            >
              <h5>
                Vendredi
              </h5>
              <h5>
                25 mai
              </h5>
            </div>
            <div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
              <div>
                <a
                  className="alert-link"
                  href="#"
                >
                  <h2
                    className="text-center"
                  >
                    <span
                      className="badge badge-secondary"
                    >
                      09:00
                    </span>
                  </h2>
                </a>
              </div>
            </div>
          </div>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
